src/main.cpp

```diff
@@ -2687,7 +2687,10 @@ bool DisconnectBlock(CBlock& block, CValidationState& state, CBlockIndex* pindex
                     LogPrint("cert", "%s():%d ApplyTxInUndo returned FALSE on tx [%s] \n", __func__, __LINE__, tx.GetHash().ToString());
                     fClean = false;
                 }
+            }
 
+            for (const CTxCeasedSidechainWithdrawalInput& cswIn:tx.GetVcswCcIn()) {
```

Comment:
move this before
`for (const CTxForwardTransferOut& fwdTransfer: tx.GetVftCcOut()) {
    if (!view.CancelSidechainEvent(fwdTransfer, pindex->nHeight)) {`
to keep exactly reversed order to ConnectBlock